ui/pages/onboarding-flow/metametrics/__snapshots__/metametrics.test.js.snap

```diff
@@ -7,17 +7,20 @@ exports[`Onboarding Metametrics Component should match snapshot 1`] = `
     data-testid="onboarding-metametrics"
   >
     <h2
-      class="box box--margin-top-1 box--margin-bottom-1 box--flex-direction-row typography typography--h2 typography--weight-bold typography--style-normal typography--align-center typography--color-text-default"
+      align="center"
```

Comment:
Seems like a prop has been incorrectly used and is being passed down to the html element